src/app.js

```diff
@@ -1,182 +1,178 @@
 import axios from 'axios';
```

Comment:
Ik had de opdracht hier op een andere manier gemaakt!
Omdat ik pas later had gelezen dat alles gesorteerd moest worden op de populatie heb ik me code aangepast en verbeterd!

____________________________________________________________________________________________________________________________
De code juiste code staat rechts! Ik heb hem geoptimaliseerd door van verschillende aspecten van de hier links staande code in functies te zetten.
Ook heb ik hem overzichtelijker gemaakt en ipv de data op basis van ingevoerde landen te pakken, pakt de uri nu de gehele database! Deze vergelijkt hij met de landen die de user in een array als argument van de getCountry functie heeft gestopt. Zo zorgt hij ervoor dat alle relevante data van de ingevoerde landen worden uitgeprint in de browser.
De methode die ik heb gebruikt om dit te bereiken is het creëren van verschillende divs d.m.v. for loops!
Ook geef ik iedere div een unieke id!

Mocht je verdere uitleg willen over hoe de code werkt?
Voel je vrij me een berichtje te sturen via teams!